convert_kinbody_to_urdf.py

```diff
@@ -0,0 +1,194 @@
+# convert_kinbody_to_urdf.py
+
+import os
+import sys
+import xml.etree.ElementTree as ET
+
+
+class Kinbody_Converter:
+    def __init__(self):
+        self.kinbody = None
+        self.urdf = None
+        self.link = None
```

Comment:
Just out of curiosity, what is the purpose of having a member variable for `link` here?